Formula/x/xcode-build-server.rb

```diff
@@ -0,0 +1,30 @@
+class XcodeBuildServer < Formula
+  desc "Build server protocol implementation for integrate xcode with sourcekit-lsp"
+  homepage "https://github.com/SolaWing/xcode-build-server"
+  url "https://github.com/SolaWing/xcode-build-server/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "90ea9b8a3e89dfa3a0bf045236c90512f5faac77b76f8039b7937bc6b34d25e0"
+  license "MIT"
+  head "https://github.com/SolaWing/xcode-build-server.git", branch: "master"
+
+  # For reading Xcode logs archives we need to unzip them
+  depends_on "gzip"
+
+  # It's a macOS only tool because it's not designed to work without Xcode
+  depends_on :macos
+
+  # Script is written in Python
+  depends_on "python@3.9"
+
+  def install
+    # Install all files and directories under libexec
+    libexec.install Dir["*"]
+
+    # Create a wrapper script in the bin directory for the xcode-build-server script
+    bin.write_exec_script libexec/"xcode-build-server"
+  end
+
+  test do
+    # Test the xcode-build-server script
+    system "#{bin}/xcode-build-server", "--help"
```

Comment:
We need a test that actually exercises some kind of functionality of the software. Version checks and calls to help rarely fail even when things aren't actually working properly,

Reply:
Script doesn't have any tests soute and I honeslty currently I don't have idea how test is properly

Reply:
That's fine if there is no other viable test option.

Reply:
Do the other LSP formula tests not work here?

Reply:
I'll take a look at it today.